change/@ni-nimble-components-2ca4a1a1-f437-4b1d-be20-f3e0e4756771.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Reminder to link to the relevant issue in the PR description. Are you going to track the 2-tone and fixed color requests separately, keep that issue around to do so, or deprioritize it and leave it untracked for now?

Reply:
👍 and tracked as #1802